rust/operator-binary/src/controller.rs

```diff
@@ -526,7 +533,7 @@ fn build_node_rolegroup_statefulset(
         .role_groups
         .get(&rolegroup_ref.role_group);
 
-    let nifi_version = nifi_version(nifi)?;
+    let nifi_version = nifi.image_version().context(NifiVersionParseFailureSnafu)?;
```

Comment:
Naming it `image_version` or `nifi_image_version` would be great

Reply:
done